source/as-promise/index.ts

```diff
@@ -75,8 +75,9 @@ export default function asPromise<T>(firstRequest?: Request): CancelableRequest<
 
 				try {
 					const hooks = options.hooks.afterResponse;
+					const hooksArray = is.array(hooks) ? hooks : [hooks];
```

Comment:
```suggestion
					const hooksArray = [hooks].flat();
```

Reply:
Is it purely for readability? One difference is `[hooks].flat()` create a shallow copy of `hooks`